processor/src/parallel/snapshots/miden_processor__parallel__tests__parallel::tests::test_trace_generation_at_fragment_boundaries::case_02.snap

```diff
@@ -2,4 +2,4 @@
66877683371826591, 1466877683371826591, 1466877683371826591, 1466877683371826591, 1466877683371826591, 1466877683371826591, 1466877683371826591, 1466877683371826591, 1466877683371826591, 1466877683371826591, 1466877683371826591, 1466877683371826591, 1466877683371826591, 1466877683371826591, 1466877683371826591, 1466877683371826591, 1466877683371826591, 1466877683371826591, 1466877683371826591, 1466877683371826591, 1466877683371826591, 1466877683371826591, 1466877683371826591, 1466877683371826591, 1466877683371826591, 1466877683371826591, 1466877683371826591, 1466877683371826591, 1466877683371826591, 1466877683371826591, 1466877683371826591, 1466877683371826591, 1466877683371826591, 1466877683371826591, 1466877683371826591, 1466877683371826591, 1466877683371826591, 1466877683371826591, 1466877683371826591, 1466877683371826591, 1466877683371826591, 1466877683371826591, 1466877683371826591, 1466877683371826591, 1466877683371826591, 1466877683371826591, 1466877683371826591, 1466877683371826591, 1466877683371826591, 1466877683371826591, 1466877683371826591, 1466877683371826591, 1466877683371826591, 1466877683371826591, 1466877683371826591, 1466877683371826591, 1466877683371826591, 1466877683371826591, 1466877683371826591, 1466877683371826591, 1466877683371826591, 1466877683371826591, 1466877683371826591, 1466877683371826591, 1466877683371826591, 1466877683371826591, 1466877683371826591, 1466877683371826591, 1466877683371826591, 1466877683371826591, 1466877683371826591, 1466877683371826591, 1466877683371826591, 1466877683371826591, 1466877683371826591, 1466877683371826591, 1466877683371826591, 1466877683371826591, 1466877683371826591, 1466877683371826591, 1466877683371826591, 1466877683371826591, 1466877683371826591, 1466877683371826591, 1466877683371826591, 1466877683371826591, 1466877683371826591, 1466877683371826591, 1466877683371826591, 1466877683371826591, 1466877683371826591, 1466877683371826591, 1466877683371826591, 1466877683371826591, 1466877683371826591, 1466877683371826591, 1466877683371826591, 1466877683371826591, 1466877683371826591, 1466877683371826591, 1466877683371826591, 1466877683371826591, 1466877683371826591, 1466877683371826591, 1466877683371826591, 1466877683371826591, 1466877683371826591, 1466877683371826591, 1466877683371826591, 1466877683371826591, 1466877683371826591, 1466877683371826591, 1466877683371826591, 1466877683371826591, 1466877683371826591, 1466877683371826591, 1466877683371826591, 1466877683371826591, 1466877683371826591, 1466877683371826591, 1466877683371826591, 1466877683371826591, 1466877683371826591, 1466877683371826591, 1466877683371826591, 1466877683371826591, 1466877683371826591, 1466877683371826591, 1466877683371826591, 1466877683371826591, 1466877683371826591, 1466877683371826591, 1466877683371826591, 1466877683371826591, 1466877683371826591, 1466877683371826591, 1466877683371826591, 1466877683371826591, 1466877683371826591, 1466877683371826591, 1466877683371826591, 1466877683371826591, 1466877683371826591,
```

Comment:
Trace length goes from 64 to 2048; is this because Plonky3 requires a trace length minimum of 2048? Is this the only thing that changed otherwise with trace generation?

Reply:
Indeed, the minimal trace length has changed to 2048. This is due to some short-comings of the current FRI implementation, which should hopefully be resolved soon.

Reply:
Also related to trace gen, we now do not insert randomness in the final NUM_RAND_ROW rows